Alpha_shapes_3/examples/Alpha_shapes_3/ex_weighted_periodic_alpha_shapes_3.cpp

```diff
@@ -41,8 +41,8 @@ int main()
   int n;
   is >> n;
   std::cout << "Reading " << n << " points " << std::endl;
-  Bare_point bp;
   for( ; n>0 ; n--) {
+    Bare_point bp;
     is >> bp;
     Weighted_point p(bp, 0.0001 * random.get_double(0., 0.015625)); // arbitrary weights
```

Comment:
The warning about "maybe undefined" will not disappear, because `is >> bp` may fail.

Reply:
I do agree that my "fix" likely won't fix anything, but it'd surprising if it were the fact that `is >> bp` can fail: we have a ton of places where we don't check for it failing (e.g. reading meshes in examples) and don't have warnings.